server/shared/types/terms/term.ts

```diff
@@ -59,29 +57,28 @@ export type GroupSetEntry = BaseGroupSet & {
 	is_subcondition?: boolean
 }
 
-export type CustomGroupSetting = {
+export type CustomQGroupSetting = {
 	/** When “predefined_groupset_idx” is undefined, will use this set of groups.
 	This is a custom set of groups either copied from predefined set, or created with UI.
 	Custom set definition is the same as a predefined set. */
 	customset: BaseGroupSet
-	disabled?: boolean
+	/** if false, not applied */
 	inuse?: boolean
```

Comment:
Should `inuse` be required?

Reply:
`inuse` isn't always present in groupsetting. For example in the categorical and single cell handlers use this code `if (!('inuse' in tw.q.groupsetting)) tw.q.groupsetting.inuse = false` because the it's potentially an empty object from the lines above.